omnigibson/robots/manipulation_robot.py

```diff
@@ -1566,7 +1566,7 @@ def teleop_data_to_action(self, teleop_action) -> th.Tensor:
         hands = ["left", "right"] if self.n_arms == 2 else ["right"]
         for i, hand in enumerate(hands):
             arm_name = self.arm_names[i]
-            arm_action = teleop_action[hand]
+            arm_action = th.tensor(teleop_action[hand]).float()
```

Comment:
nit: i think the general semantics is `.item()`? I don't quite remember, @cgokmen @ChengshuLi ?